components/transfer/style/index.less

```diff
@@ -128,8 +128,12 @@
       position: absolute;
       top: 50%;
       width: 100%;
-      margin-top: -10px;
       transform: translateY(-50%);
+
+      // with filter should offset the search box height
+      .@{transfer-prefix-cls}-list-body-with-search & {
```

Comment:
I guess this class is not added.

Reply:
Confirmed. See https://github.com/NG-ZORRO/ng-zorro-antd/blob/a2000fab4a07f6a8695922a3e35e1620ff87d6e2/components/transfer/nz-transfer-list.component.html#L31.

No classed are binded.

Reply:
I'll fix it.